lib/chef/provider/package/cab.rb

```diff
@@ -45,7 +45,7 @@ def remove_package(name, version)
         end
 
         def dism_command(command)
-          shellout = Mixlib::ShellOut.new("dism.exe /Online #{command} /NoRestart", { :timeout => @new_resource.timeout })
+          shellout = Mixlib::ShellOut.new("dism.exe /Online /English #{command} /NoRestart", { :timeout => @new_resource.timeout })
```

Comment:
You can use that switch, but the existing Mixlib::ShellOut.new call is wrong and needs changing as well